src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
```

Comment:
According to the checklist, you shouldn't use tag names for styling (except `html` and `body`). It's recommended to use class names instead. This way, you can avoid applying styles to undesired elements and improve the reusability of your styles.